Source/JavaScriptCore/dfg/DFGSpeculativeJIT.cpp

```diff
@@ -1073,7 +1082,10 @@ void SpeculativeJIT::compileGetById(Node* node, AccessType accessType)
 
         base.use();
 
-        cachedGetById(node->origin.semantic, baseRegs, resultRegs, stubInfoGPR, scratchGPR, node->cacheableIdentifier(), JITCompiler::Jump(), NeedToSpill, accessType);
+        JSValueRegsTemporary property(this);
+        fillPropertyRegs(m_jit, node->cacheableIdentifier(), property);
+
+        cachedGetById(node->origin.semantic, baseRegs, property.regs(), resultRegs, stubInfoGPR, scratchGPR, node->cacheableIdentifier(), JITCompiler::Jump(), NeedToSpill, accessType);
```

Comment:
These changes are regressing code generation for 64bit environment. Instead of passing JSValueReg, we should pass appropriate EncodedJSValue in cachedGetById. So, let's revert `DFGSpeculativeJIT.cpp` change.